IssueTracker/IssueTracker.FileSystem/Repository/IRepository/IFileProvider.cs

```diff
@@ -3,5 +3,5 @@ public interface IFileProvider
 {
     Task UploadAsync(Models.File file);
```

Comment:
are Models.File and IssueTracker.FileSystem.Models.File the same dto?

Reply:
no